LayoutTests/imported/w3c/web-platform-tests/cookie-store/cookieStore_get_arguments.https.any-expected.txt

```diff
@@ -6,10 +6,6 @@ FAIL cookieStore.get with name in options promise_test: Unhandled rejection with
 FAIL cookieStore.get with name in both positional arguments and options promise_test: Unhandled rejection with value: object "NotSupportedError: The operation is not supported."
 FAIL cookieStore.get with absolute url in options promise_test: Unhandled rejection with value: object "NotSupportedError: The operation is not supported."
 FAIL cookieStore.get with relative url in options promise_test: Unhandled rejection with value: object "NotSupportedError: The operation is not supported."
-FAIL cookieStore.get with invalid url path in options promise_rejects_js: function "function() { throw e }" threw object "NotSupportedError: The operation is not supported." ("NotSupportedError") expected instance of function "function TypeError() {
-    [native code]
-}" ("TypeError")
-FAIL cookieStore.get with invalid url host in options promise_rejects_js: function "function() { throw e }" threw object "NotSupportedError: The operation is not supported." ("NotSupportedError") expected instance of function "function TypeError() {
-    [native code]
-}" ("TypeError")
+FAIL cookieStore.get with invalid url path in options assert_unreached: Should have rejected: undefined Reached unreachable code
+FAIL cookieStore.get with invalid url host in options assert_unreached: Should have rejected: undefined Reached unreachable code
```

Comment:
I'm not sure that this change should have happened, but I put it here to get a second pair of eyes on it.